src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.FSharp.DesignTime.targets

```diff
@@ -14,6 +14,7 @@
     <AppDesignerFolder Condition="'$(AppDesignerFolder)' == ''">Properties</AppDesignerFolder>
     <AppDesignerFolderContentsVisibleOnlyInShowAllFiles Condition="'$(AppDesignerFolderContentsVisibleOnlyInShowAllFiles)' == ''">false</AppDesignerFolderContentsVisibleOnlyInShowAllFiles>
     <LanguageServiceName Condition="'$(LanguageServiceName)' == ''">F#</LanguageServiceName>
+    <LanguageServiceId Condition="'$(LanguageServiceId)'==''">{BC6DD5A5-D4D6-4DAB-A00D-A51242DBAF1B}</LanguageServiceId>
```

Comment:
Sucks that we have to have these duplicated :(

Reply:
LanguageServiceId & Name? If only Roslyn used the SID...